src/app/main/component/user/components/habit/add-new-habit/add-new-habit.component.ts

```diff
@@ -91,6 +93,10 @@ export class AddNewHabitComponent implements OnInit, OnDestroy {
       });
   }
 
+  public getProgressValue(value: number): void {
+    this.canAcquire = value >= 80;
```

Comment:
would be better to make a constant for `80` to make it understandable

Reply:
You're right, I will add it